dot/core/service.go

```diff
@@ -482,14 +482,9 @@ func (s *Service) ProcessBlockAnnounceMessage(msg network.Message) error {
 		return err
 	}
 
-	_, err = s.blockState.GetBlockData(header.Hash())
-	if err != nil && err.Error() == "Key not found" {
-		// send block request message
-		log.Debug("[core] sending new block to syncer", "number", blockAnnounceMessage.Number)
-		s.blockNumOut <- blockAnnounceMessage.Number
-	} else if err != nil {
-		return err
-	}
```

Comment:
why is this logic removed? we should keep it, but change `GetBlockData` to `GetBlock`

Reply:
done